### ISSUE TITLE
Some links of the github page are dead

### ISSUE THREAD

**Issue by Ricola3D** [SoftwareBrothers/adminjs#874]
In the github project page, some of the links are dead : 
- https://adminjs-example-app-staging.herokuapp.com/admin
- documentation (https://adminjs.com/docs)
- live demo (https://adminjs-example-app-staging.herokuapp.com/admin)
- join the community (https://adminjs.slack.com/join/shared_invite/zt-djsqxxpz-_YCS8UMtQ9Ade6DPuLR7Zw)

**Comment by dziraf** [comment]
@Ricola3D thank you for reporting this. I've already updated the links.

We've recently rebranded from AdminBro to AdminJS so parts of readme/docs might still be outdated. We're also still choosing the new logo so our documentation still uses old assets. We're also still using the old Slack because migrating this is a little bit more complicated.

The correct links are:

Demo - https://demo.adminjs.co/admin
Documentation - https://adminjs.co (or https://docs.adminjs.co)
Slack - https://join.slack.com/t/AdminBro/shared_invite/zt-djsqxxpz-_YCS8UMtQ9Ade6DPuLR7Zw
